docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -8,55 +8,71 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 include::_attributes.adoc[]
 :categories: security,web
 
-Quarkus has an integrated pluggable web security layer. If security is enabled all HTTP requests will have a permission
+Quarkus has an integrated pluggable web security layer. If security is enabled, all HTTP requests will have a permission
 check performed to make sure they are allowed to continue.
 
-NOTE: Configuration authorization checks are executed before any annotation-based authorization check is done, so both
-checks have to pass for a request to be allowed. This means you cannot use `@PermitAll` to open up a path if the path has
-been blocked using `quarkus.http.auth.` configuration. If you are using JAX-RS you may want to consider using the
-`quarkus.security.jaxrs.deny-unannotated-endpoints` or `quarkus.security.jaxrs.default-roles-allowed` to set default security
-requirements instead of HTTP path level matching, as these properties can be overridden by annotations on an individual
-endpoint.
+[NOTE]
+====
+Configuration authorization checks are executed before any annotation-based authorization check is done, so both
```

Comment:
It looks like an important piece of information to retain

Reply:
@jherrman  Both passive voices are caused by the absence of the word Quarkus, which executes and doing the check done. But since this is a Quarkus documentation, maybe it is not necessary.